packages/manager/apps/telecom/src/app/telecom/pack/migration/offers/pack-migration-offers.controller.js

```diff
@@ -71,7 +78,38 @@ export default class TelecomPackMigrationOffersCtrl {
   }
 
   selectOffer(offer) {
-    this.TucPackMigrationProcess.selectOffer(offer);
+    const selectedOffer = offer;
+    const params = {
+      offerName: selectedOffer.offerName,
+    };
+
+    // Retrieve option values (included and added)
+    const options = Object.entries(selectedOffer.options)
```

Comment:
If no use of key for each entry:
```suggestion
    const options = Object.values(selectedOffer.options)
```